Qast/Managers/LocationManager.swift

```diff
@@ -9,19 +9,26 @@
 import Foundation
 import CoreLocation
 import Mapbox
+import SDWebImage
 
 protocol LocationManagerDelegate: class {
     var mapView: MGLMapView { get }
     
     func qastMap(didReceive nearbySoundZones: [SoundZone])
     func qastMap(didUpdate currentSoundZone: SoundZone?)
     func qastMap(didUpdate userLocation: CLLocation)
+    
+    // once we use our our MGLCalloutView, this method will move to MapViewController
+    // this is not location-related IMO
+    // mere product of LocationManager being mapView delegate to access the user's location...
+    func qastMap(didTap soundZone: SoundZone)
 }
 
 class LocationManager: NSObject, MGLMapViewDelegate {
     
     weak var delegate: LocationManagerDelegate?
     var networker: SoundZoneAPI = FirebaseManager()
+    var streamManager = StreamManager()
```

Comment:
It seems like fetching metadata from UMG could fall under SoundZoneAPI or another protocol called StreamAPI this way we don't need to have the stream manager here

Reply:
Ya I think so too. gonna move it out